service/synapse-service-rest/src/main/java/io/americanexpress/synapse/service/rest/model/ServiceRouting.java

```diff
@@ -44,17 +44,17 @@ public ServiceRouting() {
      * @return the client identifier.
      */
     @ApiModelProperty(value = "Unique client identifier assigned to the consumer")
```

Comment:
should we change here to say id as well? this is just a question, not sure.

Reply:
Done